test/e2e/model/user_input.go

```diff
@@ -0,0 +1,59 @@
+package model
+
+import (
+	// "os"
+	"path/filepath"
+
+	// . "github.com/onsi/gomega"
+	// . "github.com/onsi/gomega/gstruct"
+	// "go.mongodb.org/atlas/mongodbatlas"
+
+	// kube "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kube"
+	// mongocli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/mongocli"
+	. "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/config"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+// TODO PRIVATE?
+type UserInputs struct {
```

Comment:
small recommendation: try doing refactoring separately from adding features - this will make reviewing a bit easier and PRs smaller